pkgs/tools/misc/mbuffer/default.nix

```
@@ -25,7 +25,7 @@ stdenv.mkDerivation rec {
   meta = {
     homepage = "http://www.maier-komor.de/mbuffer.html";
     description  = "A tool for buffering data streams with a large set of unique features";
-    license = lib.licenses.gpl3;
+    license = lib.licenses.gpl3Only;
```

The file headers say `GPLv3+`.

```suggestion
    license = lib.licenses.gpl3Plus;
```

---

Nice catch, though the README states gpl3Only:
```
LICENSE:
========
GNU GPLv3 (see file LICENSE for details)
```
The LICENSE file is just a copy of GPL3.
I'll try to contact the author to clarify this.

---

The license file is the same for both variants. Later also include a ``or at your option any later version`` in the source files.

---

The response from Thomas is that he wants to check newer versions before accepting them, so gpl3Only is the correct choice here.